src/lookup-table/models/LookupTable.model.ts

```diff
@@ -158,7 +159,7 @@ const LookupTableModel = () => {
     tableName: string,
     fieldMaps: FieldMapType[]
   ): Promise<boolean> => {
-    const text = `UPDATE data_table SET field_map = $1 WHERE display_name = $2`
+    const text = `UPDATE data_table SET field_map = $1 WHERE table_name = $2`
```

Comment:
This was an existing mistake -- the field maps weren't getting updated when new columns were added due to this.